dist/default-build-script/Assets/Editor/UnityBuilderAction/Builder.cs

```diff
@@ -56,27 +56,37 @@ public static void BuildProject()
       // of either `UnityEditor.BuildPlayerOptions` or `UnityEditor.BuildPlayerWithProfileOptions`
       dynamic buildPlayerOptions;
 
-      if (options["customBuildProfile"] != "") {
+      if (options.TryGetValue("activeBuildProfile", out var buildProfilePath) && !string.IsNullOrEmpty(buildProfilePath)) {
```

Comment:
Shall we go with a descriptive error, in case of null or empty?

Reply:
commited a fix